sources/packages/api/src/app.module.ts

```diff
@@ -166,6 +169,9 @@ import { AppSystemAccessModule } from "./app.system-access.module";
     SINValidationService,
     DesignationAgreementLocationService,
     StudentAssessmentService,
+    StudentScholasticStandingsService,
```

Comment:
Are these ones needed in the main module? I was under the impression that they would be consumed only under the specific modules. Is there any controller/service here that needs them?

Reply:
I was also under the same impression,  but during development, API was throwing me an import error in appModules, that is why I added it. But now, after seeing this comment, I removed and tested it's not complaining about the import, I will update the code. Thanks for that